install/install.qmd

```diff
@@ -89,6 +89,12 @@ For more information, see the
 [CmdStanR documentation](https://mc-stan.org/cmdstanr/articles/cmdstanr.html#installing-cmdstan){target="_blank"}
 :::::
 
+::::: {.install #install-cmdstanr-conda .hidden}
+Run `conda install -c conda-forge r-cmdstanr`.
+
+Note: this will also install CmdStan and any system prerequesites.
+:::::
+
 ::::: {.install #install-cmdstanr-github-src .hidden}
 In R, run `remotes::install_github("stan-dev/cmdstanr")`.
```

Comment:
instead of using remotes and stan-dev github, instructions should be to r-universe.

Reply:
R-universe is the option two above this one